app/scenes/Document/components/Document.js

```diff
@@ -91,7 +92,7 @@ class DocumentScene extends React.Component<Props> {
     } else if (prevProps.document.revision !== this.lastRevision) {
       if (auth.user && document.updatedBy.id !== auth.user.id) {
         this.props.toasts.showToast(
-          `Document updated by ${document.updatedBy.name}`,
+          t(`Document updated by ${document.updatedBy.name}`),
```

Comment:
```suggestion
          t(`Document updated by {{userName}}`, { userName: document.updatedBy.name }),
```

Good catch, but gotta do the interpolation right 😉

Reply:
Ohh, sorry about that!

Reply:
I wrote it in on GitHub so sorry about the linting